templates/model/goldentest/v1beta3/upgrade.go

```diff
@@ -16,15 +16,21 @@ package goldentest
 
 import (
 	"context"
+	"fmt"
+
+	"github.com/jinzhu/copier"
 
 	"github.com/abcxyz/abc/templates/model"
-	"github.com/abcxyz/pkg/logging"
+	v1beta4 "github.com/abcxyz/abc/templates/model/goldentest/v1beta4"
 )
 
 // Upgrade implements model.ValidatorUpgrader.
 func (t *Test) Upgrade(ctx context.Context) (model.ValidatorUpgrader, error) {
-	logger := logging.FromContext(ctx).With("logger", "Upgrade")
-	logger.DebugContext(ctx, "finished upgrading goldentest model, this is the most recent version")
+	var out v1beta4.Test
+
+	if err := copier.Copy(&out, t); err != nil {
```

Comment:
I think we need to do something like `SkipStdout` here to make sure that a new binary doesn't attempt to verify the stdout of old tests. Though probably you have this planned for a separate PR.

Reply:
I will do it in another PR. This PR focuses on gitkeep ut and goldentest upgrade